### PR TITLE
Add Occitan language

### PR DESCRIPTION
**Pull request by Quenty31**
### This PR will... add the Occitan language to the interface.

### CODE REVIEW

File: src/assets/translations/oc.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    "exitFullscreen": "Sortir del Mòde Ecran complèt",
+    "fullscreen": "Ecran complèt",
+    "hd": "Qualitat",
+    "liveBroadcast": "Dirècte",
+    "logo": "Logo",
+    "mute": "Copar lo Son",
+    "next": "Seguent",
+    "nextUp": "Vidèo venenta",
+    "notLive": "Fòra dirècte",
+    "off": "Dsactivar",
```
Comment (karimMourra) [inline src/assets/translations/oc.json]: @Quenty31  should this be Desactivar ? it seems like a letter might be missing after the 'D'

Comment (Quenty31) [reply src/assets/translations/oc.json]: Yes, sorry!

**Comment by karimMourra** [comment]
@Quenty31 could you please try rebasing your branch? The integration tests failed to run

**Comment by Quenty31** [comment]
@karimMourra don't really know how to =/

**Comment by aliciahurst** [comment]
Hi @Quenty31, we're translating some more fields in the player (the keyboard shortcuts modal) for an upcoming release. Are you interested in providing the Occitan for those? If so, let me know what your email address is, or shoot me one at alicia@jwplayer.com. Thanks so much!

**Comment by Quenty31** [comment]
Sent!

**Pull request closed without merging**